cypress/support/step-definitions/applitools-steps.js

```diff
@@ -0,0 +1,11 @@
+When('Element displays correctly', () => {
```

Comment:
This step is used as Then in bdd
```suggestion
Then('Element displays correctly', () => {
```